README.md

```diff
@@ -192,13 +195,17 @@ df = pd.read_csv("my_dataset.csv")
 #### Modin is a DataFrame designed for datasets from 1MB to 1TB+ 
 
 We have focused heavily on bridging the solutions between DataFrames for small data 
-(e.g. pandas) and large data. Often data scientists require different tools for doing
-the same thing on different sizes of data. The DataFrame solutions that exist for 1KB do
-not scale to 1TB+, and the overheads of the solutions for 1TB+ are too costly for 
-datasets in the 1KB range. With Modin, because of its light-weight, robust, and scalable
-nature, you get a fast DataFrame at small and large data. With preliminary [cluster](https://modin.readthedocs.io/en/latest/using_modin.html#using-modin-on-a-cluster)
-and [out of core](https://modin.readthedocs.io/en/latest/out_of_core.html)
-support, Modin is a DataFrame library with great single-node performance and high
+(e.g. pandas) and large data. Often data scientists have to use different tools 
```

Comment:
"bridging the solutions between" is incorrect usage of "bridge" and I don't think we mean to say that there are other "solutions between". I suggest replacing the first sentence with

```
We have focused heavily on bridging the gap between Dataframes for small data
(e.g. regular pandas) and for large data (e.g. Dask and Vaex?? does Vaex have overhead? Or are we thinking of e.g. using Spark. but Spark doesn't usually have dataframes, except for things like [Pandas on Spark](https://databricks.com/blog/2021/10/04/pandas-api-on-upcoming-apache-spark-3-2.html)).
```

note that I'm not sure which solutions for large data to include here.

Reply:
@devin-petersohn Any thoughts here?

Reply:
Agree with @mvashishtha

Reply:
I actually prefer on this particular part to drop the first sentence entirely and starting with "Often data scientists...". I think that giving an example of Dataframe for large data with Dask/Vaex/Pandas on Spark gives those solutions free advertising and we already have a section for comparing Dask Dataframe vs. Koalas vs. Modin in the FAQ. @mvashishtha thoughts?

Reply:
I like that idea! Your rationale sounds good.